bot-brain/data-collector/preprocess.js

```diff
@@ -0,0 +1,51 @@
+const natural = require('natural');
+const stopword = require('stopword');
+const fs = require("fs")
+const lodash = require('lodash');
+
+const rawArticleData = require('../public/articles.json');
+
+function preprocessForEmbedding(){
+
+    const preprocessArticles = (articles) => {
+    const tokenizer = new natural.WordTokenizer();
+    const stemmer = natural.PorterStemmer;
+
+    return articles.map(article => {
+        const preprocessedTextContents = article.textContents.map(text => {
+        // Convert to lowercase
+        let cleanedText = text.toLowerCase();
+
+        // Tokenize the text
+        let tokens = tokenizer.tokenize(cleanedText);
+
+        // Remove stop words
+        //   tokens = stopword.removeStopwords(tokens);
+
+        // Stem the words
+        //   tokens = tokens.map(token => stemmer.stem(token));
+
+        // Rejoin tokens into a cleaned string
+        cleanedText = tokens.join(' ');
```

Comment:
what's the use of this var `cleanedText`?

Reply:
Prev written some other logic, later I remove but missed to remove this `var`. No use.